exercises/practice/minesweeper/.meta/Example.roc

```diff
@@ -0,0 +1,45 @@
+module [annotate]
+
+isBomb : List Str, I64, I64 -> Result Bool [OutOfBounds]
+isBomb = \rows, nx, ny ->
+    if nx < 0 || ny < 0 then
+        Err OutOfBounds
+    else
+        rowChars = rows |> List.get? (Num.toU64 ny) |> Str.toUtf8
+        rowChars |> List.get? (Num.toU64 nx) |> Bool.isEq '*' |> Ok
```

Comment:
```suggestion
isBomb : List Str, I64, I64 -> Result Bool [OutOfBounds]
isBomb = \rows, nx, ny ->
    y = Num.toU64Checked? ny
    x = Num.toU64Checked? nx
    rows
    |> List.get? y
    |> Str.toUtf8
    |> List.get? x
    |> Bool.isEq '*'
    |> Ok
```
How about this instead? I originally had x and y inlined but that causes a compiler crash so I extracted them. Bug opened here: https://github.com/roc-lang/roc/issues/7049.